kolibri/content/models.py

```diff
@@ -109,53 +109,60 @@ class Admin:
     def __str__(self):
         return self.title
 
-class MimeType(AbstractContent):
-    """
-    Normalize the "kind"(mimetype) of Format model
-    """
-    readable_name = models.CharField(max_length=50)
-    machine_name = models.CharField(max_length=100)
+class ContentKind(AbstractContent):
+    kind = models.CharField(max_length=200, blank=True, null=True)
```

Comment:
This should probably just be the primary key, and have `choices` that draw from the list of constants. (As in the `auth` app).